.github/workflows/test.yml

```diff
@@ -153,7 +154,7 @@ jobs:
     with:
       repo: oclif/plugin-plugins
       os: ${{ matrix.os }}
-      command: 'yarn test:integration'
+      command: yarn test:integration --retries 3
```

Comment:
suggestion: `external-test` could use the normal retries code
salesforcecli/github-workflows/.github/actions/retry instead of all of its consumers need to do this

Reply:
Hm, actually. I don't think this will work since `external-test.yml` uses `working-directory`. An external action (`uses`) needs to support passing it in, which the retry action we use does not https://github.com/nick-fields/retry

https://stackoverflow.com/questions/58139175/running-actions-in-another-directory#comment126980684_63122434

Reply:
@iowillhoit @mshanemc I don't think we can use the retry action since you need to be able to set the `working_directory`, which as far as I can tell isn't possible with that action

Reply:
haha exactly 😅 We we're both looking at this at the same time ^